### DISCUSSION TITLE
Twitch stream not detected

### DISCUSSION THREAD

**Discussion by Ch01ch** [aclap-dev/video-downloadhelper#986] [Help and Q&A]
### Your question

Hello, for several days now VideoDownHelper has no longer detected streams on Twitch. I use vdh version 9.0.2.12 target google, locale fr. on Chrome, Windows 10 and the Companion app (v2.0.19). I disabled all the extensions in case one of them conflicted, I restarted VDH, Chrome, but the problem persists. Does anyone have a solution to my problem? Thanks in advance.

### Addon & CoApp version and details

version: 9.0.2.12
target: google
channel: stable
lang: fr
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>"
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": false,
    "open_count_store": 26,
    "successfull_dl": 9
}

**Comment by GrampaWildWilly** [comment]
This appears to be a duplicate of #959.  You should read that one.

**Reply by Ch01ch** [reply]
Hello GrampaWildWilly and thank you for your response. Indeed, my problem is identical to post #959, same symptoms, I do not use any antivirus other than that of Windows 10, I have a VPN but even uninstalling the problem persists, I deactivated the firewalls but nothing changes. I'm still looking for the solution :) Thanks.

**Reply by Mattmjs** [reply]
I have the extension in Ungoogled Chromium and I have detection. Try the plain vanilla chromium browser and see if you
have detection. If you do then it must be a bug in Chrome. There was previously a bug in the Firefox code where there was no detection on Twitch. That took some time for Mozilla to resolve.

**Comment by paulrouget** [comment]
@Ch01ch can you share the url of one of the stream that's not working for you? Thanks. I'll look into it.

**Reply by Ch01ch** [reply]
Hi Paul, this is Twitch       https://www.twitch.tv/

**Reply by Ch01ch** [reply]
None of the streams on this platform are detected

**Reply by GrampaWildWilly** [reply]
Paul, when you go to the Twitch home page, there is usually at the top of the page some livestream that appears to be randomly selected.  When I've gone to that home page, VDH always detects something that I can download.  I don't usually let it download for more than a minute or so because that content is of utterly no interest to me.  But I have always been successful getting that default livestream.  Of course, I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3880, Firefox 128.0 64-bit, licensed VDH 9.0.2.14a1 beta, CoApp 2.0.19, which appears to not be the same as what the folks having trouble are running.  You don't need a specific URL.  Just go to the home page & something will be there.

**Reply by paulrouget** [reply]
Sorry. I do need a specific URL. Just share one that's not working. (please don't reply "any urls". I really do need one specific URLs).

**Reply by Ch01ch** [reply]
Hello Paul, these are all the urls of this platform, but I will give you some random url of this platform: https://www.twitch.tv/alisa, https://www.twitch .tv/zulzorander, https://www.twitch.tv/clemovitch ...

**Reply by GrampaWildWilly** [reply]
Your second URL has an erroneous blank between `twitch` & `.tv` but I think we can deal with that.  Those are all archived livestreams.  What's the term for livestreams that aren't live any more?  Deadstreams?  Whatever.  I was able to download the first little bit of each of these & they played fine.  That's not much of a surprise.  We've already concluded VDH for Firefox has no trouble with Twitch.

Paul, you don't need specific URLs.  No, you don't.  If somebody posts the URL of a livestream here, it will be a deadstream by the time you get here.  Just go to the home page of the web site.  There's always a default livestream at the top of the page.  Any one will do.  They're all the same from the perspective of the VDH software.  Plus there's a bunch of other livestreams listed on the rest of the page if you feel like testing with more than one sample.  I have never visited Twitch except to investigate an issue reported here by a fellow user.  So I could be wrong, but it is my impression that the primary focus of Twitch is livestreams.  Whatever investigating you're doing, you need to make sure there's no surprises between livestreams & deadstreams.  Break down & just go to the home page of the site & make sure that at least the default livestream of the moment also works.  That's in addition to the deadstreams our buddy @Ch01ch has posted.

**Reply by Ch01ch** [reply]
GrampaWildWilly , Paul . I thank you for the time you spend resolving my problem, but the simplest thing is that I will re-license under Firefox and perhaps one day following a Chrome update this will be resolved.

**Reply by paulrouget** [reply]
> you don't need specific URLs

I do. Twitch doesn't always use the same system depending on the country, link, browser and publisher.

@Ch01ch thanks for the info. I'll investigate.

**Reply by GrampaWildWilly** [reply]
> depending on the country, link, browser and publisher

Oh brother.  Could it be any more complicated?  You mean in some countries, the Twitch home page does NOT have a default livestream at the top of the page?

**Comment by paulrouget** [comment]
Should be working now.
